debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresConnection.java

```diff
@@ -510,6 +510,7 @@ public synchronized void close() {
      * @throws SQLException if anything fails.
      */
     public Long currentTransactionId() throws SQLException {
+        /* 
```

Comment:
Same as above, lets add a YB Note explaining why this doesn't matter. Some sort of info would be good for future reference.

Reply:
At present, it appears that this txnid information is not being used to make any material difference.